src/libs/OptionsListUtils.ts

```diff
@@ -2301,7 +2301,7 @@ function getHeaderMessageForNonUserList(hasSelectableOptions: boolean, searchVal
  * Helper method to check whether an option can show tooltip or not
  */
 function shouldOptionShowTooltip(option: ReportUtils.OptionData): boolean {
-    return (!option.isChatRoom || !!option.isThread) && !option.isArchivedRoom;
+    return (!option.isChatRoom || !!option.isThread) && !ReportUtils.isArchivedRoom(option, ReportUtils.getReportNameValuePairs(option?.reportID));
```

Comment:
Can we change [this](https://github.com/Expensify/App/blob/main/src/libs/ReportUtils.ts#L428) from isArchivedRoom to private_isArchived. And then we can directly use option.private_isArchived.

Reply:
Hm, I think we're actually trying to work towards standardizing using private_isArchived. I think the entire point of the option is to have data about the report. The Report data type already has private_isArchived so all we would have to do is remove this [line](https://github.com/Expensify/App/blob/main/src/libs/ReportUtils.ts#L428) which unnecessarily declares a different variable and then use the current variable (private_isArchived) which has the right data.

Reply:
We're in the process of moving some keys from the `reports` collection to a new collection named `reportNameValuePairs` which is why this is currently in the codebase. When we do that, we will most likely need to create a new `reportNameValuePairs` data type similar to the reports data type and move all related keys (along with the private_isArchived key that we will change to using in this PR).